src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/Envelope.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.entity;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.UUID;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import static java.util.Collections.emptyList;
+
+@Entity
+@Table(name = "envelopes")
+public class Envelope {
+
+    @Id
+    private UUID id;
+
+    @JsonProperty("po_box")
+    private String poBox;
```

Comment:
what do those annotations do on private fields?

Reply:
Works on private fields as well. Is not part of the work but used all annotations to prove locally the parsing does what it does. Simply left as a no harmer since proxy (?) object is already with json property constructors. Getters and setters will simply fall into same json object